user_interaction.py

```diff
@@ -1,110 +1,98 @@
 #!/usr/bin/env python
```

Comment:
What is the difference between this script and the one in magenta_docker directory?